tests/unit/Change.spec.ts

```diff
@@ -286,40 +289,41 @@ describe('Change component', () => {
     expect(state.businessInformation.identifier).toBe('FM1234567')
 
     // Validate Office Addresses
-    expect(state.officeAddresses.deliveryAddress.streetAddress)
-      .toBe('reg delivery_address - address line one')
-    expect(state.officeAddresses.mailingAddress.streetAddress)
-      .toBe('rec mailing_address - address line two')
+    // expect(state.officeAddresses.deliveryAddress.streetAddress)
+    //   .toBe('reg delivery_address - address line one')
+    // expect(state.officeAddresses.mailingAddress.streetAddress)
+    //   .toBe('rec mailing_address - address line two')
```

Comment:
Needs to be fixed.

Reply:
Future unit test. The data structure is wrong

Reply:
![image](https://user-images.githubusercontent.com/3484109/231244951-d7de3df8-4d95-447d-a859-e6336472cfb1.png)

Reply:
Awesome, thanks!

Reply:
The take-away here is that typing in the unit tests is nice to have?